Makefile

```diff
@@ -1,9 +1,9 @@
 .PHONY: lint test coverage
 
 lint:
-	python -m pylint check_brevisone.py
```

Comment:
Can we maybe make this variable? Since `python` will hopefully mean python3 in most distros one day

Reply:
Not sure if they are going to fall for the same thing twice :-)
My problem is, for now, `python` is not there on debian (since I don't have python2 anymore).
In any case I doubt that `python3` will be deprecated, since it is the more explicit naming anyway and that avoids errors in any direction.

But if you insist or don't feel convinced I will pack all of that in a variable

Reply:
Not sure either what's the best solution. But we should use something that works for all our Python repos.

On my Ubuntu 20.04 `python` and `python3` are the same. Personally I would prefer simply `python` and the virtualenv should decide which version is behind that

Reply:
Not sure if this compatible to not-`python3` anymore. I mean those are similar but different environments.

Reply:
Or: https://packages.debian.org/bullseye/python-is-python3